mr/src/test/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandlerTestUtils.java

```diff
@@ -61,6 +62,7 @@ static TestHiveShell shell() {
     TestHiveShell shell = new TestHiveShell();
     shell.setHiveConfValue("hive.notification.event.poll.interval", "-1");
     shell.setHiveConfValue("hive.tez.exec.print.summary", "true");
+    shell.setHiveConfValue(OrcConf.FORCE_POSITIONAL_EVOLUTION.getHiveConfName(), "true");
```

Comment:
Can you add a comment for this that explains why it is set for the Hive tests? I think it could be confusing later that this is explicitly set to true, only to be overridden by ORC when creating a reader.

Reply:
Good point!
Added the comment